src/rest.rs

```diff
@@ -303,6 +301,14 @@ fn handle_request(req: Request<Body>, query: &Arc<Query>, cache: &Arc<Mutex<LruC
             let value = attach_tx_data(value, network, query);
             json_response(value)
         },
+        (&Method::GET, Some(&"tx"), Some(hash), Some(&"hex")) => {
+            let hash = Sha256dHash::from_hex(hash)?;
+            if let Some(rawtx) = query.txstore_get_raw(&hash) {
```

Comment:
Why not a match?

Reply:
No real reason, it was originally structured differently without the `else` branch which made this seem more readable. I'll force-push an update.